exporter/streams.go

```diff
@@ -4,22 +4,26 @@ import (
 	"github.com/gomodule/redigo/redis"
 	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
+	"strconv"
```

Comment:
Can you group and sort the stdlib imports separately please? 
[This is a good example ](https://github.com/oliver006/redis_exporter/blob/master/main.go#L3-L17)if what it should look like.

Note to self: the linter should catch this, not sure why it didn't, I need to check what's up and fix it.

Reply:
Sure, this wasn't changed intentionally. It made my local linter edit on save. My first GO project so I still working on tuning my dev env.